tests/Organization.py

```diff
@@ -741,6 +741,13 @@ def testRemoveCustomProperty(self):
         with self.assertRaises(github.UnknownObjectException):
             self.org.get_custom_property("property_1")
 
+    def testGetSelfHostedRunners(self):
+        # XXX: Organization BeaverSoftware does not exist
+        # https://api.github.com/orgs/BeaverSoftware
+        org = self.g.get_organization("ClickHouse")
+        runners = org.get_self_hosted_runners()
+        self.assertEqual(runners.totalCount, 602)
+
```

Comment:
Contributors collect test data from their org and then rename it to "BeaverSoftware" to mimic a single org.
```suggestion
        runners = self.org.get_self_hosted_runners()
        self.assertEqual(runners.totalCount, 602)

```